apps/web/lib/response/utils.ts

```diff
@@ -726,9 +780,6 @@ export const getResponseMeta = (
 
     responses.forEach((response) => {
       Object.entries(response.meta).forEach(([key, value]) => {
-        // skip url
-        if (key === "url") return;
-
```

Comment:
If we remove this, the function will start collecting values for the URL field, but we won’t use them in the select dropdown. I recommend returning early for URL fields with an empty array. 

<img width="1114" height="734" alt="Image" src="https://github.com/user-attachments/assets/839338e6-d0d2-4ec7-9389-c7b8cf266783" />